scripts/gen-service-kustomize.sh

```diff
@@ -504,6 +504,15 @@ if [[ "${KIND}" == "OpenStackControlPlane" ]]; then
   patch: |-
     - op: remove
       path: /spec/dns/template/options/0/values/0
+EOF
+    else
+        cat <<EOF >>kustomization.yaml
+- target:
+    kind: ${KIND}
+  patch: |-
+    - op: replace
+      path: /spec/dns/template/options/0/values/0
```

Comment:
do we need a way to not set this this? It's not a required field IIUC

Reply:
From chatting with @abays: the sample CRs typically set this to 192.168.122.1, so this is normally a no-op.

However the replace op would fail if a sample CR is used that does not set it, which is addressed in the latest commit.

Reply:
Also the docs explicitly set 2 upstream DNS servers so I don't think we need to consider the case where this isn't set and DHCP is providing it.